README.rst

```diff
@@ -0,0 +1,152 @@
+===========================
+openwrt-openwisp-monitoring
+===========================
+
+.. image:: http://img.shields.io/github/release/openwisp/openwrt-openwisp-monitoring.svg
+   :target: https://github.com/openwisp/openwrt-openwisp-monitoring/releases
+
+.. image:: https://img.shields.io/gitter/room/nwjs/nw.js.svg?style=flat-square
+   :target: https://gitter.im/openwisp/general
+   :alt: support chat
+
+------------
+
+`OpenWRT <https://openwrt.org/>`_ configuration agent for 
+`OpenWISP Monitoring <https://github.com/openwisp/openwisp-monitoring>`_.
+
+**Want to help OpenWISP?** `Find out how to help us grow here
+<http://openwisp.io/docs/general/help-us.html>`_.
+
+.. image:: http://netjsonconfig.openwisp.org/en/latest/_images/openwisp.org.svg
+  :target: http://openwisp.org
+
+.. contents:: **Table of Contents**:
+ :backlinks: none
+ :depth: 3
+
+Configuration options
+---------------------
+
+UCI configuration options must go in ``/etc/config/monitoring``.
+
+- ``monitored_interfaces``: interfaces that needs to be monitored, defaults to ``*`` for all interfaces.
+- ``interval``: time after which device data should be sent to server, defaults to ``300``
+
+Compiling openwrt-openwisp-monitoring
+-------------------------------------
+
+There are 2 packages for *openwisp-netjson-monitoring*:
+
+- **netjson-monitoring**: provides NetJSON Device Monitoring output
+- **openwisp-monitoring**: depends on **netjson-monitoring** and **openwisp-config**
+
+There are four variants of openwisp-monitoring:
+
+- **openwisp-monitoring-openssl**: depends on *openwisp-config-openssl* and *netjson-monitoring*
+- **openwisp-monitoring-mbedtls**: depends on *openwisp-config-mbedtls* and *netjson-monitoring*
+- **openwisp-monitoring-wolfssl**: depends on *openwisp-config-wolfssl* and *netjson-monitoring*
+- **openwisp-monitoring-nossl**: depends on *openwisp-config-nossl* and *netjson-monitoring*
+
+The following procedure illustrates how to compile both *openwisp-monitoring* and *netjson-monitoring*:
+
+.. code-block:: shell
+
+    git clone https://github.com/openwrt/openwrt.git openwrt
+    cd openwrt
+    git checkout openwrt-19.07
+
+    # configure feeds
+    echo "src-git monitoring https://github.com/openwisp/openwrt-openwisp-monitoring.git" > feeds.conf
+    cat feeds.conf.default >> feeds.conf
+    ./scripts/feeds update -a
+    ./scripts/feeds install -a
+    # any arch/target is fine because the package is architecture indipendent
+    arch="ar71xx"
+    echo "CONFIG_TARGET_$arch=y" > .config
+    echo "CONFIG_PACKAGE_rpcd=y" >> .config
```

Comment:
Why is this needed? If it's a dependency, it should be defined in the `Makefile` and hence will not be needed here, it will be added automatically by `make defconfig`.